### DISCUSSION TITLE
Where do we track issues

### DISCUSSION THREAD

**Discussion by alexschroeter** [arkitektio/arkitektio.github.io#24] [General]
I think, especially for new users, it is impossible to decide in which repository they should submit an issue to. For us, tracking issues over all the different repositories might also become a burden.

I am wondering if we should dedicate one specific repo to be the front-facing one where issues are tracked. Something like arkitektio/arkitektio. 
- we could disable issues in the others (or maybe hide them and only use internally [I would probably prefer disable])
- we should link to this one in the documentation and organization description

What do you think?

**Comment by jhnnsrs** [comment]
Thats a great idea i think. Not sure how this is handled by other organizations? 

I believe its good to have the issues in the others, as once pull requests come in we can easily track issues in the
corresponding repos.

Then we could decide where to put the corresponding issues (just by transfering the issues).

**Comment by alexschroeter** [comment]
I took a look at the status quo for other projects, and this is what I found:

- I found this old (pre projects) reddit thread which talks about this https://www.reddit.com/r/github/comments/ssr38y/how_do_companies_do_organization_wide_issues_and/?rdt=35589 but there exists a github action https://github.com/actions/add-to-project
  - TLDR: Projects will solve this (imho it didn't), use external tool, create a dedicated repo.

- GitHub has published the projects feature, which is supposed to give developers a way to handle this kind of things.
  - So I took a look at it, and it turns out that the most basic feature "automatically adding issues" to a project is working and can be setup for multiple repos from the Projects page.

Since we don't have that much traffic yet and highest priority issue we should solve is **confusion on where new users should post/find issues**. I think we should not set up a JIRA or different tool at this time and revisit when it becomes necessary. I would propose creating an arkitektio/arkitektio repository which will house all general issues regarding Arkitekt and link in these issues to their relevant PR/Issues in other repositories if necessary. And set up a Projects page for us so we don't have to look in all repositories for issues.

**Reply by alexschroeter** [reply]
To see a bit of an example of this you can check out this Project https://github.com/orgs/arkitektio/projects/5 which is setup to auto import new issues and prs from kabinet and orkestrator-next as examples.
